apps/earn-protocol-landing-page/components/layout/LandingPageContent/LandingPageContent.tsx

```
@@ -0,0 +1,78 @@
+        <Text
+          as="h1"
+          variant="h1"
+          style={{ color: 'var(--earn-protocol-primary-100)', textAlign: 'center' }}
+        >
+          Highest Quality Yield
+        </Text>
+      </div>
+      <Carousel
+        components={content.map((item) => (
+          <LandingPageStrategyPicker {...item} />
+        ))}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)